pkg/proxy/iptables/proxier.go

```diff
@@ -1108,6 +1108,18 @@ func (proxier *Proxier) syncProxyRules() {
 				// Currently we only create it for loadbalancers (#33586).
 				writeLine(natRules, append(args, "-j", string(svcXlbChain))...)
 			}
+
```

Comment:
We could do the "no endpoints" check before writing the MARK/SVC or XLB rules.  But I presume we want to keep the rules less volatile?